cli/fathom_web/list.py

```diff
@@ -0,0 +1,32 @@
+from click import argument, command, File, option, Path
+import pathlib
+
+
+@command()
+@option('--out_file', '-o', type=File(mode='w'), default=None,
+        help='A file for saving the printed filenames for easy future reference.')
+@argument('in_directory', type=Path(exists=True, file_okay=False, allow_dash=True))
```

Comment:
I'm not sure that allow_dash is applicable for a dir.

Reply:
As we spoke about, I will remove it.